package.json

```
@@ -1,7 +1,8 @@
 {
   "name": "replit",
   "displayName": "replit",
-  "description": "",
+  "description": "VS Code extension for connecting to repls using the crosis api",
```

```suggestion
  "description": "VS Code extension for connecting to repls",
```

Should we mention crosis? Seems like an implementation detail that the user doesn't need to know about. Then again you need an API key to use this.

---

Hmmm, true. Might be better not to say, but up to you.

---

I'll go ahead, and get rid of it, but we can change it later